release-notes/9.0/os-packages.json

```diff
@@ -134,7 +134,7 @@
                         },
                         {
                             "id": "krb5",
-                            "name": "libgssapi-krb5-2"
+                            "name": "krb5"
```

Comment:
This will require regeneration of the markdown.

Reply:
Good call!